joomla/joomla-cms
# Wrong message shown when saving profile in backend

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/28934.

### Summary of Changes

Corrects message shown when saving profile in backend.

### Testing Instructions

1) Edit your profile in backend (using `com_admin`). Save it. Check the message shown.
2) Send a private message. Check the message shown.

### Expected result

1) > Profile saved.
2) > Message sent.

### Actual result

1) > Message sent.
2) > Message sent.

### Documentation Changes Required

No.
